workflows/rnaseq/main.nf

```diff
@@ -870,11 +708,14 @@ workflow RNASEQ {
         // Provide MultiQC with rename patterns to ensure it uses sample names
         // for single-techrep samples not processed by CAT_FASTQ.
 
-        ch_name_replacements = ch_fastq.single
+        ch_name_replacements = ch_fastq
+            .filter{ meta, reads ->
+                reads.size() == 1
+            }
             .map{ meta, reads ->
-                def name1 = file(reads[0]).simpleName + "\t" + meta.id + '_1'
+                def name1 = file(reads[0][0]).simpleName + "\t" + meta.id + '_1'
                 if (reads[1] ){
-                    def name2 = file(reads[1]).simpleName + "\t" + meta.id + '_2'
+                    def name2 = file(reads[0][1]).simpleName + "\t" + meta.id + '_2'
                     return [ name1, name2 ]
```

Comment:
What's this whole change about?

Reply:
It's because I'm using ch_fastq directly (since the one with `single` and `multiple` branches is moved to the subworkflow). So these are now tuples output by groupTuple requiring the double index:

```
[[foo.fastq.gz,bar.fastq.gz]]
```